04_numeronym/joshcarp/neumonym_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var stdout = os.Stdout
+
+var tests = []struct {
+	input    []string
+	expected []string
+}{
+	{[]string{"accessibility", "Kubernetes", "abc"}, []string{"a11y", "K8s", "abc"}},
```

Comment:
Please consider non-invalid user input such as non alphanumeric charcters. E.g. '!nternational' should not be a valid input. "Web Develpment" should yield 'W12t" and not 'W13t' with spaces not being counted (according to wiki page). Similarly '12345' should not be give the result of '135' as it is not a word to start off with.

Reply:
Ok so now it only shortens if the word is of alpha characters, otherwise it just returns the original string. 
Didn't consider this point before, 
Thanks!